### PR TITLE
Update doc about installation for raspi

### PR DESCRIPTION
**Pull request by asukiaaa**
### Contributor checklist

- [ x] I have read the CONTRIBUTING.rst
- [ x] I have tested my contribution on these devices: Epson TM-T88IV
- [ x] My contribution is ready to be merged as is

----------

### Description
I simplified doc for raspi to use pip3.

### CODE REVIEW

File: doc/user/raspi.rst [hunk 1]
```diff
@@ -2,95 +2,41 @@
 Raspberry Pi
 ************
 
-:Last Reviewed: 2016-12-07
+:Last Reviewed: 2016-12-24
 
-This instructions were tested on Raspbian.
+This instructions were tested on Raspbian Jessie.
 
-.. warning:: You should **never** directly connect an printer with RS232-interface (serial port) directly to
```
Comment (patkan) [inline doc/user/raspi.rst]: Is there a special reason that you have changed this warning box to a normal chapter?

**Comment by asukiaaa** [comment]
Thank you for feedback.

> Is there a special reason that you have changed this warning box to a normal chapter?

Sorry, I did not find that warning syntax is active on the document because I checked syntax on github.
I agree to keep this warning box so I rolled back it.

references
- [github page](https://github.com/python-escpos/python-escpos/blob/development/doc/user/raspi.rst)
- [doc page](https://python-escpos.readthedocs.io/en/latest/user/raspi.html)

> We should probably keep it because people might find it more easily but maybe we should just link to the installation manual.

I added a link of installation-manual.

How is it?
Thanks.

**Comment by patkan** [comment]
Thank you for your changes. This will be merged.

**Comment by asukiaaa** [comment]
Thank you for merging!

**Comment by patkan** [comment]
You are welcome! Just one question: we plan on changing the license in the future (I  have not yet asked everyone that contributed). Would MIT be OK for you?

**Comment by asukiaaa** [comment]
I'm OK. Thanks.

**Pull request merged**